CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 Changelog
 =========
 
+[1.8.0] - 2022-05-09
+--------------------
+
+### Changes
+
+- Support playbooks which use `gather_facts: false`
```

Comment:
this seems like a feature to me, too. Or a bug fix. Not a Change.

Reply:
Updated, thanks.